commands/deploy.go

```diff
@@ -697,3 +729,36 @@ func findMissingValues(signingDetails common.SigningDetails) string {
 	}
 	return strings.Join(missingValues, ",")
 }
+
+func getRepositoryNamePrefix(registry string) (string, error) {
```

Comment:
I'm not sure that this fn is needed, if you use `getRepositoryName` as suggested elsewhere.